src/extensions/nativeExtensions/image/ImageExtensionActionRenderBtn.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <v-dialog v-model="dialog" max-width="500px">
+    <template v-slot:activator="{ on: onDialog }">
```

Comment:
```suggestion
    <template #activator="{ on: onDialog }">
```
Can we use new slot syntax? And in other places too.